src/sentry/integrations/jira/client.py

```diff
@@ -127,13 +127,13 @@ def update_comment(self, issue_key, comment_id, comment):
     def get_projects_list(self):
         return self.get_cached(self.PROJECT_URL)
 
-    def get_project_key_for_id(self, project_id):
+    def get_project_key_for_id(self, project_id) -> str:
```

Comment:
Shouldn't `project_id` have a type?

Reply:
it should, but this isn't being type checked yet and I don't know what the type is

Reply:
👍